### PR TITLE
Ddpb 2224 costs checklist

### PR DESCRIPTION
**Pull request by digitalali-moj**

### CODE REVIEW

File: src/AppBundle/Resources/views/Admin/Client/Report/partials/_prof-deputy-costs.html.twig [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                    {{ macros.summaryTableHeaderQA(hideEditLink) }}
+                    <tbody>                
+                    {% include 'AppBundle:Report/ProfDeputyCosts:_how_charged_answers.html.twig' with {
+                        'report': report,
+                        'transDomain': 'report-prof-deputy-costs',
+                        'hideEditLink' : true
+                    } %}
+                    {% include 'AppBundle:Report/ProfDeputyCosts:_all_cost_total_answers.html.twig' with {
+                        'report': report,
+                        'submittedOtherCosts': report.generateActualSubmittedOtherCosts,
```
Comment (aiesoftware) [inline src/AppBundle/Resources/views/Admin/Client/Report/partials/_prof-deputy-costs.html.twig]: This calculation is being performed several times, can we `set actualOtherCosts = ...`?

File: src/AppBundle/Resources/translations/admin-checklist.en.yml [hunk 2]
```diff
@@ -55,6 +56,8 @@ checklistPage:
     giftsSummary: View gifts summary
     deputyExpensesSummary: View deputy expenses
     lifestyleResponses: View health and lifestyle responses
+    profDeputyCosts: View deputies responses
```
Comment (aiesoftware) [inline src/AppBundle/Resources/translations/admin-checklist.en.yml]: Should this be 'deputy's', similar to line 125 below? (I'm aware the design has 'deputies').

Comment (slizzio) [reply src/AppBundle/Resources/translations/admin-checklist.en.yml]: yes think you're right, good spot

File: src/AppBundle/Resources/views/Admin/Client/Report/partials/_prof-deputy-costs.html.twig (hunk 1 above)
Comment (slizzio) [reply src/AppBundle/Resources/views/Admin/Client/Report/partials/_prof-deputy-costs.html.twig]: I'll look into this

**Comment by slizzio** [comment]
Needs to merge into feature-105 branch

**Pull request closed without merging**